mesheryctl/internal/cli/root/system/provider.go

```diff
@@ -305,7 +305,7 @@ mesheryctl system provider switch [provider]
 var resetProviderCmd = &cobra.Command{
 	Use:   "reset",
 	Short: "reset provider to default",
-	Long:  "Reset provider of context in focus to default (No provider set)",
+	Long:  "Reset provider of context in focus to default (Meshery)",
```

Comment:
Above language seems confusing.

“Reset provider for current context to default (Meshery)”.

Reply:
Changed

Reply:
@abdullah1308 I meant `context in focus` ----> `current context`
`Long:  "Reset provider of current context to default (Meshery)",`

Reply:
Done